src/application.ts

```diff
@@ -33,6 +33,8 @@ export interface Options {
   octokit?: InstanceType<typeof ProbotOctokit>;
   throttleOptions?: any;
   webhooks?: Webhooks;
+  port?: number;
+  host?: string;
```

Comment:
I don't think these options make sense for application, the server is only started in `src/index.ts`.

I know `src/index.ts` and `src/application.ts` are rather confusing, I want to merge the two eventually and get rid of all the duplication.

Reply:
Ok, looks like removing those won't break tests either, sending new patch